src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -154,6 +158,8 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
         },
         [policyID],
     );
+    const sortedTags = lodashSortBy(Object.values(policyTagLists.at(0)?.tags ?? {}), 'name', localeCompare) as PolicyTag[];
```

Comment:
Revert this to be back inside the below `useMemo`. And for the `selectedTagsObject` you can use `policyTagLists.at(0)?.tags` (the tags don't need to be sorted for that use).